src/material.h

```diff
@@ -161,6 +161,9 @@ class Material: public Resource {
   /// type minimum and maximum mass criteria.
   virtual void ChangePackageId(int new_package_id = Package::unpackaged_id());
 
+  /// Packages the material
+  virtual std::vector<Material::Ptr> Package(Package::Ptr pkg);
```

Comment:
Looks like this needs to be:
```suggestion
  virtual std::vector<Resource::Ptr> Package(Package::Ptr pkg);
```